src/localization/locales/en-US.json

```diff
@@ -293,7 +293,14 @@
         "list_map_help_url": "https://terraso.org/help/add-a-new-landscape-to-terraso/",
         "list_map_section_label": "Landscapes map",
         "list_map_popup_link": "View details about {{name}}",
-        "boundaries_update_save": "Update Map"
+        "boundaries_update_save": "Update Map",
+        "form_boundary_draw_polygon_title": "[TODO] Draw {{name}}’s Boundary",
+        "form_boundary_draw_polygon_description": "[TODO] Click anywhere to start drawing. Save your drawing and return to complete later. You can adjust the boundary in edit mode. ",
```

Comment:
issue: this is not accurate. You can't click anywhere. You need to click the polygon tool first. It's also ungrammatical. Needs complete rewrite.